src/abstractdataframe/io.jl

```diff
@@ -94,6 +94,7 @@ writetable(filename, df, [keyword options])
 * `separator::Char` -- The separator character that you would like to use. Defaults to the output of `getseparator(filename)`, which uses commas for files that end in `.csv`, tabs for files that end in `.tsv` and a single space for files that end in `.wsv`.
 * `quotemark::Char` -- The character used to delimit string fields. Defaults to `'"'`.
 * `header::Bool` -- Should the file contain a header that specifies the column names from `df`. Defaults to `true`.
+* `append::Bool` -- If `filename` already exists, the dataframe is only appended to it. `append` = true turns automatically sets `header` = false but it makes sure that the names of the DataFrame being appended to the file correspond to the header in the file.
```

Comment:
No, being explicit sounds fine to me. Though there's a typo in "turns automatically sets", and backticks should be around the whole `append = true` and  `header = false` expressions.